### PR TITLE
Filter

### PR DESCRIPTION
**Pull request by thaistcosta**

### CODE REVIEW

**Comment by thaistcosta** [comment]
version working... except project

**Review by lexarisa — APPROVED** [review]
<3 <3 very nice work

File: client/common/components/BoardSection.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ import styles from '../../styles/BoardSection.module.css';
 import { useRouter } from 'next/router';
 import { useAppDispatch, useAppSelector } from '../store/hooks/redux-hooks';
 import { setCurrentTask } from '../store/slices/task/task.slices';
+import { IoIosAdd, IoIosClose } from 'react-icons/io';
```
Comment (alexandra-plana) [inline client/common/components/BoardSection.tsx:11]: 🪀

File: client/common/components/MainDashBoard.tsx [hunk 2]
```diff
@@ -18,20 +18,22 @@ const MainDashboard = () => {
 
   useEffect(() => {
     milestoneEvent();
-  }, []);
```
Comment (alexandra-plana) [inline client/common/components/MainDashBoard.tsx:21]: why wouldn't we have the [ ] there?

File: client/pages/project/[token].tsx [hunk 3]
```diff
@@ -1,68 +1,99 @@
...
-                      <p>{el.project.description}</p>
-                      <p>{el.project.status}</p>
-                    </div>
-                  </Link>
-                </div>
-              ))}
-          </div>
+    <div className={styles.container}>
+      <div className={styles.cardWrapper}>
+        <h1>Welcome to Alena, {data.username}</h1>
```
Comment (alexandra-plana) [inline client/pages/project/[token].tsx:62]: ee

**Review by alexandra-plana — APPROVED** [review]
so impressive!!!!!!!!!!! 🍅

**Comment by thaistcosta** [comment]
anything

**Pull request merged**